k3s.yaml

```diff
@@ -231,9 +231,13 @@ subpackages:
 
 update:
   enabled: true
+  # Ignore any release-candidate tags
+  ignore-regex-patterns:
+    - -rc*
+  # Example: Upstream tag: v1.31.0+k3s1 would become v1.31.0.1
   version-transform:
-    - match: \.(\d+)$
-      replace: +k3s$1
+    - match: \+k3s(\d+)$
```

Comment:
Looks like this was never working. It was copy/pasted from the transforms logic futher up, which was ADDING back the k3s for the git clone. Here we need to remove it.